doc/classes/CanvasItem.xml

```diff
@@ -405,6 +405,19 @@
 				Returns the [RID] of the [World2D] canvas where this item is in.
 			</description>
 		</method>
+		<method name="get_canvas_layer" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the layer number of the [CanvasLayer] where the node is in.
+				If the node is not in any [CanvasLayer], the layer number will be `0`.
+			</description>
+		</method>
+		<method name="get_canvas_layer_node" qualifiers="const">
+			<return type="CanvasLayer" />
+			<description>
+				Returns the [CanvasLayer] where the node is in, or null if the node is not in any [CanvasLayer].
```

Comment:
```suggestion
				Returns the [CanvasLayer] that this node is in, or [code]null[/code] if the node is not in any [CanvasLayer].
```

Reply:
We tend to avoid prepositions after the subject even if it's perfectly correct English because it's harder to read for non-native speakers _(speaking from experience writing these, not actually a maintainer)_, so the latter may be more appropriate.

If you want convey the relation between the two more, instead of using "of" which is pretty generic, you could say for example "Returns the [CanvasLayer] that contains this node".